stock_cycle_count/models/stock_cycle_count.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2017-18 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockCycleCount(models.Model):
+    _name = "stock.cycle.count"
+    _description = "Stock Cycle Counts"
+    _inherit = "mail.thread"
+    _order = "id desc"
+
+    @api.model
+    def _default_company(self):
+        company_id = self.env["res.company"]._company_default_get(self._name)
+        return company_id
```

Comment:
This is deprecated, just add `default=lambda self: self.env.company` in the field.